res/schema.xml

```diff
@@ -582,4 +582,18 @@ reapplying those migrations.
       UPDATE library SET filetype='aiff' WHERE filetype='aif';
     </sql>
   </revision>
+  <revision version="40" min_compatible="3">
+    <description>
+      Add Macros
+    </description>
+    <sql>
+      CREATE TABLE IF NOT EXISTS macros (
```

Comment:
- slot INTEGER NOT NULL
- A UNIQUEness constraint for (track_id, slot) is missing